apb/defaults/main.yml

```diff
@@ -44,7 +44,7 @@ broker_refresh_interval: "600s"
 broker_launch_apb_on_bind: "false"
 broker_output_request: "true"
 broker_recovery: "true"
-broker_auto_escalate: ""
+broker_auto_escalate: "true"
```

Comment:
Before we merge this I would like to ensure team wants this to 'true' for upstream.
I saw the downstream patch is setting it to 'false'.

In general I don't like deviating behavior from downstream/upstream unless we have a clear reason for doing so.

CC:  @jmrodri @shawn-hurley @eriknelson @rthallisey

Reply:
Just to add the reason why I am defaulting upstream to false revolves around the fact that the service catalog has the origin identity header behind a feature gate. This feature gate is not enabled by default in their helm chart. Furthermore, the service catalog has no interrogation mechanism (ie. Ask if that feature is enabled).

Reply:
Just to add the reason why I am defaulting upstream to false revolves around the fact that the service catalog has the origin identity header behind a feature gate. This feature gate is not enabled by default in their helm chart. Furthermore, the service catalog has no interrogation mechanism (ie. Ask if that feature is enabled).

Reply:
Maybe we just document that if Service Catalog does not have the feature gate enabled they MUST edit the configmap to enable auto-escalate.

Reply:
I think if we really want to leave auto_escalate set to false, then we update the log messages to say something like "hey, we didn't get an origin identity header. Either enable it in the service catalog or set auto_escalate to true in the broker config."